goss/goss-windows.yaml

```diff
@@ -62,7 +62,7 @@ command:
     exit-status: 0
     stdout:
       - /16\.\d+\.\d+\.\d+/
-    skip: {{ not (eq .Env.AGENT_OS_VERSION "2019") }}
+    skip: {? {not (eq .Env.AGENT_OS_VERSION "2019"): ''} : ''}
```

Comment:
Ping @smerle33 it looks like updatecli for python3 is doing some weird things here, like the ruby one :'(

Reply:
Ref https://github.com/jenkins-infra/packer-images/commit/d6c776f1b942fd13d5b4fa54ce324baacb00484c